spark/pyspark/Makefile

```diff
@@ -1,14 +1,20 @@
-container_name=pubnative/pyspark-ci
-version=1.0.2
-tag=$(container_name):$(version)
-
+NAME 	:= pubnative/pyspark-ci
+TAG 	:= $$(git log -1 --pretty=%h)
+IMG 	:= ${NAME}:${TAG}
+LATEST 	:= ${NAME}:latest
+ 
 all: build push
 
-build:
-	docker build . -t $(tag)
+compile_dependencies:
+	@pip-compile \
+	  --generate-hashes \
+	  --output-file requirements.txt \
+	  requirements.in
+	@chmod +r requirements.txt
```

Comment:
@ValentinMouret Looks professional. You don't want to see commands in the output?

Reply:
I'm all about professionalism 🤓